app/views/articles/show.html.erb

```diff
@@ -231,8 +231,13 @@
   <% end %>
 <% end %>
 
+<div class="mod-actions-menu hidden"><iframe src="<%= "#{@article.path}/mod" %>"></iframe></div>
```

Comment:
This should probably also be inside the conditional since we probably don't even want it in the DOM for non-trusted users.

Reply:
yup, it did cross my mind 🤔 will do